distribution/lib/Standard/Database/0.0.0-dev/src/Connection/Connection_Options.enso

```diff
@@ -12,3 +17,16 @@ type Connection_Options
     merge :  Vector -> Vector
     merge self base_options =
         base_options.filter x->(self.options.any (y->y.first==x.first) . not) + self.options
+
+    ## PRIVATE
+       Widget to design connection options.
+    default_widget : Widget
+    default_widget =
+        fqn = Meta.get_qualified_type_name Pair . drop (..Last 5)
```

Comment:
I think I'd prefer to use
```suggestion
        fqn = Meta.get_qualified_type_name Pair . drop (..After_Last ".")
```
but that is quite subjective.

Reply:
Will tweak in my next PR.